scripts/globals/moghouse.lua

```diff
@@ -42,8 +57,38 @@ function moogleTrigger(player,npc)
             end
         end
 
+        local homeNationFame = player:getFame(player:getNation())
```

Comment:
Everything looks good to me save for this should be `getFameLevel`~!

The [comment for `player:getFame()` is a bit confusing in that it says "fame level"](https://github.com/DarkstarProject/darkstar/blob/master/src/map/lua/lua_baseentity.cpp#L5546), but it's actually returning raw fame.

You'll [want `'player:getFameLevel()'`](https://github.com/DarkstarProject/darkstar/blob/master/src/map/lua/lua_baseentity.cpp#L5751) because it [converts the player's raw fame into the 1-9 integers](https://github.com/DarkstarProject/darkstar/blob/master/src/map/lua/lua_baseentity.cpp#L5773) used for fame checks.